app.rb

```diff
@@ -0,0 +1,101 @@
+require_relative 'book'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+require_relative 'classroom'
+
+class App
+  def initialize()
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_all_books
+    @books.each do |book|
+      puts "Title: \"#{book.title}\", Author: #{book.author}"
+    end
+  end
+
+  def list_all_people
+    @people.each do |person|
+      puts "[#{person.type}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
+
+  def create_a_person(choice, name, age)
+    if choice == '1'
+      person = create_a_student(name, age)
+      person.type = 'Student'
+    elsif choice == '2'
+      person = create_a_teacher(name, age)
```

Comment:
- [ ] I also cannot create a teacher. Kindly run your app from beginning to end and ensure it works.

  ![image](https://github.com/waelelsafty07/oop-school/assets/36542986/af248b57-c711-41e7-ae02-7e2ee0c24412)